sympy/core/power.py

```diff
@@ -459,6 +462,9 @@ def _eval_is_negative(self):
         elif self.base.is_positive:
             if self.exp.is_real:
                 return False
+        elif self.base.is_zero:
+            if self.exp.is_real:
+                return self.exp.is_zero and False
```

Comment:
do you mean simply: `elif self.base.is_zero: return False`? It will never be negative. A test would be
```python
>>> (Dummy(zero=True)**Dummy()).is_negative
False
```

Reply:
I don't think this should be the case:
```py
>>> (0**zoo).is_negative  # nan.is_negative
>>>
```
But thanks I think I can simplify it more to the following
```
elif self.base.is_zero:
    return self.exp.is_real and False
```

Reply:
`return self.exp.is_real and False` will always be False

Reply:
so you could do
```
elif self.base.is_zero:
    if self.exp.is_real:
        return False
```

Reply:
or 
```python
elif fuzzy_and(self.base.is_zero, self.exp.is_real):
    return False
```

Reply:
Actually Pythonically `True and False => False, False and False => False, None and False => None`.
But after thinking more I got that the result should be
`return not self.exp.is_real and None`, it would result `False` when `self.exp.is_real` is `True` and `None` in the other cases.

Reply:
Or I can just use
```py
if self.exp.is_real:
    return False
```

Reply:
> so you could do
> 
> ```python
> elif self.base.is_zero:
>     if self.exp.is_real:
>         return False
> ```
This is enough and I would change it so.

Reply:
Thanks @smichr for pointing out.